sections/03_categorize.md

```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
+
+*Lack of true labels is perhaps among the biggest obstacles for electronic 
+health records(EHR)-based learing studies (e.g., phenotyping). Popular deep 
+learning (and machine learning) methods are mostly classification problem, 
+which require ground-truth labels (i.e., response variables) for training. 
+For EHR, unfortunately, this means we need to hire multiple expert clinicians 
+to manually read and annotate individual patients' records to assign "true" 
+labels (i.e., expert chart review). Sometimes, the learnt features also need to 
+be manually validated and interpreted by clinicians. This can be quite costly 
+in time and money [doi: 10.1016/j.ijmedinf.2016.09.014]. Due to resource limitations, 
+many existing works in this direction (including many referenced in this review) 
+skipped the expert review process, which may face great skepticism from 
+medical domain experts. To date, even the most resourceful large national consortia 
+have difficulty acquiring enough expert-validated labeled data. For instance, 
+in the eMERGE consortia and PheKB database [@url: https://phekb.org/implementations], 
+most sample sizes with expert validations are between 100 to 300 patients, 
+which is quite small even for simple machine learning, not to mentioned data-hungry 
+deep learning models.*
+
+*This issue also common to many related domains -- around medical images, omics. 
+(thus the motivation for semi-supervised learning as briefly mentioned in earlier sections)*
+
+*In summary, this subsection emphasize on two important aspects that are not 
+addressed by other sections: 
+1) Data (and sometimes learnt features/representations) 
```

Comment:
In imaging, we are starting to see papers where groups have made investments in physician time to acquire labeled training data (e.g. #151, #207).  Does that fit here or are we discussing that elsewhere (I haven't read all the related pull requests).

To @cgreene's point, I think this a very interesting topic.  Somewhere (Twitter I guess?) I've seen discussion about data versus methods and how companies may be releasing their deep learning methods as open source code but keeping their valuable datasets private.